Cannot install goapp command

When doing `go get -u -v github.com/murlokswarm/app/...`, the installing of the `goapp` command fails with the following error

```
# github.com/murlokswarm/app/cmd/goapp
./exec.go:46:4: undefined: app.Error
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -16,7 +16,7 @@ script:
   - go test ./bridge
   - go test ./html
   - go test ./file
-  - go test ./cmd/goapp                                                                                                                             
+  - go build ./cmd/goapp                                                                                                                             
   - goveralls -service travis-ci -repotoken $COVERALLS_TOKEN -coverprofile cover.out
 
 notifications:

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,9 +4,8 @@ go:
   - 1.10.x
 
 install:
-  - go get -v -t .
+  - go get -v -t ./...                                                                                        
   - go get -v github.com/mattn/goveralls
-  - go get -v github.com/stretchr/testify
   - go install -v github.com/mattn/goveralls
 
 script:
@@ -15,7 +14,7 @@ script:
   - go test ./bridge
   - go test ./html
   - go test ./file
-  - go test ./cmd/goapp
+  - go test ./cmd/goapp                                                                                                                             
   - goveralls -service travis-ci -repotoken $COVERALLS_TOKEN -coverprofile cover.out
 
 notifications:

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,6 +15,7 @@ script:
   - go test ./bridge
   - go test ./html
   - go test ./file
+  - go test ./cmd/goapp
   - goveralls -service travis-ci -repotoken $COVERALLS_TOKEN -coverprofile cover.out
 
 notifications:

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ go:
   - 1.10.x
 
 install:
-  - go get -v -t .                                                                                        
+  - go get -v -t .
   - go get -v github.com/mattn/goveralls
   - go install -v github.com/mattn/goveralls
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,8 +4,10 @@ go:
   - 1.10.x
 
 install:
-  - go get -v -t ./...                                                                                        
+  - go get -v -t .                                                                                        
   - go get -v github.com/mattn/goveralls
+  - go get -v github.com/disintegration/imaging
+  - go get -v github.com/segmentio/conf
   - go install -v github.com/mattn/goveralls
 
 script:

--- a/.travis.yml
+++ b/.travis.yml
@@ -6,8 +6,6 @@ go:
 install:
   - go get -v -t .                                                                                        
   - go get -v github.com/mattn/goveralls
-  - go get -v github.com/disintegration/imaging
-  - go get -v github.com/segmentio/conf
   - go install -v github.com/mattn/goveralls
 
 script:
@@ -16,7 +14,6 @@ script:
   - go test ./bridge
   - go test ./html
   - go test ./file
-  - go build ./cmd/goapp                                                                                                                             
   - goveralls -service travis-ci -repotoken $COVERALLS_TOKEN -coverprofile cover.out
 
 notifications:

--- a/cmd/goapp/exec.go
+++ b/cmd/goapp/exec.go
@@ -5,8 +5,6 @@ import (
 	"io"
 	"os"
 	"os/exec"
-
-	"github.com/murlokswarm/app"
 )
 
 func execute(cmd string, args ...string) error {
@@ -43,7 +41,7 @@ func printOutput(r io.Reader, output io.Writer) {
 			return
 		}
 		if err != nil {
-			app.Error(err)
+			printErr("%s", err)
 			continue
 		}
 		output.Write(b[:n])

